### PR TITLE
add sample tsv to data import modal

### PR DESCRIPTION
**Pull request by Shakespeared**
Adds a downloadable sample TSV file and a link to a terra support page from the 'import table data' modal on the data page. 

Also adds mixpanel tracking for downloading the example TSV and for uploading data through the modal. 

Tested in the UI and verified events in mixpanel.

### CODE REVIEW

File: src/components/data/data-utils.js [hunk 1]
```diff
@@ -212,6 +213,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       const workspace = Ajax().Workspaces.workspace(namespace, name)
       await (useFireCloudDataModel ? workspace.importEntitiesFile : workspace.importFlexibleEntitiesFile)(file)
       onSuccess()
+      Ajax().Metrics.captureEvent(Events.workspaceDataUpload, {
```
Comment (ansingh7115) [inline src/components/data/data-utils.js:216]: Intellij gives me a warning here: "Missing await for an async function call "

File: src/components/data/data-utils.js [hunk 2]
```diff
@@ -212,6 +213,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
...
+        workspaceNamespace: namespace, workspaceName: name
+      })
```
Comment (davidangb) [inline src/components/data/data-utils.js:218]: noting that we only log this event if the upload was successful. I assume that's your intention!

**Review by davidangb — APPROVED** [review]
lgtm! one minor comment inline, not a blocker

File: src/components/data/data-utils.js (hunk 2 above)
Comment (Shakespeared) [reply src/components/data/data-utils.js:218]: @davidangb yes, that was my intention! If you disagree with that implementation I can change it, but I figured we wouldn't want to duplicate events if someone tries to upload a bunch of times with something not formatted correctly, etc.

File: src/components/data/data-utils.js (hunk 1 above)
Comment (Shakespeared) [reply src/components/data/data-utils.js:216]: Yeah, that's on most of the mixpanel implementations, it's a known condition but isn't an issue. It happens when the mixpanel call is inside an async function even though the mixpanel event itself doesn't need to be awaited.

File: src/components/data/data-utils.js (hunk 2 above)
Comment (davidangb) [reply src/components/data/data-utils.js:218]: that works for me! If our business folks decide they want to track failures too, we can always change it later.

**Review by sagehen03 — APPROVED** [review]
:+1:

**Pull request merged**